blockchain/v2/io.go

```diff
@@ -48,7 +48,13 @@ type consensusReactor interface {
 }
 
 func (sio *switchIO) sendBlockRequest(peer p2p.Peer, height int64) error {
-	msgBytes, err := bc.EncodeMsg(&bcproto.BlockRequest{Height: height})
+	msgProto := new(bcproto.Message)
+
+	if err := msgProto.Wrap(&bcproto.BlockRequest{Height: height}); err != nil {
```

Comment:
Why not just construct `bcproto.Message` structs directly? I.e. `msgProto := &bcProto.Message{Sum: &bcproto.BlockRequest{Height: height}}`.

Reply:
We could, they're semantically equivalent.

Reply:
Yes, but it's much shorter and doesn't need error handling.